ansible/roles/pgo-operator/defaults/main.yml

```diff
@@ -23,11 +23,14 @@ crunchy_debug: "false"
 enable_crunchyadm: "false"
 
 pgo_client_install: "true"
+pgo_client_container_install: "false"
 pgo_cluster_admin: "false"
 pgo_disable_tls: "false"
 pgo_tls_no_verify: "false"
 pgo_disable_eventing: "false"
 pgo_apiserver_port: 8443
+pgo_apiserver_url: "https://postgres-operator"
```

Comment:
would a better default be `https://localhost:8443`?

Reply:
I tried setting the env var in the container to `https://localhost:8443` and it didn't connect
```
Error: Get https://localhost:8443/version: dial tcp [::1]:8443: connect: connection refused
```

I originally set it to `postgres-operator` so that it would be able connect using the service as long as its in the same namespace.

Reply:
Ah, makes sense. Carry on. :wink: :laughing: